README.md

```diff
@@ -26,7 +26,7 @@ Multimodal Classification for Analysing Social Media -- [Summary](https://medium
 
 ## Dialogue and Interactive Systems
 
-Towards a Human-like Open-Domain Chatbot — [Summary](https://ai.googleblog.com/2020/01/towards-conversational-agent-that-can.html), [Paper](https://arxiv.org/abs/2001.09977)
+Towards a Human-like Open-Domain Chatbot — [Summary](https://ai.googleblog.com/2020/01/towards-conversational-agent-that-can.html), [Paper](https://arxiv.org/abs/2001.09977), [Video](https://www.youtube.com/watch?v=STrrlLG15OY&t=225s)
```

Comment:
The video looks great! Just a small comment to update the link without the timing encoded. :)